modules/openapi-generator/src/main/resources/Java/libraries/webclient/ApiClient.mustache

```diff
@@ -128,6 +129,8 @@ public class ApiClient {
     */
     public static WebClient buildWebClient(ObjectMapper mapper, DateFormat dateFormat) {
         mapper.setDateFormat(dateFormat);
+        mapper.registerModule(new JavaTimeModule());
+        mapper.findAndRegisterModules();
```

Comment:
Are you sure you need to do this ? We don't do it for the other jackson-based generators

Reply:
Hello @cbornet, I pushed a repro scenario here: https://github.com/marcoreni/openapi-generator/tree/repro/910 

You can see the issue by launching samples\client\echo\java\issue910\src\test\java\org\openapitools\client\api\Issue910Test

Reply:
No I mean we register the `JavaTimeModule` but we don't call `findAndRegisterModules`

Reply:
No, that's not mandatory. I added it because it would allow to autoregister other modules that may be added in the future.

Reply:
If it's not mandatory I would remove it. And give the possibility to customize the ObjectMapper to the client if needed.

Reply:
OK, I'll push a fix soon. Do you think that `mapper.registerModule()` should be applied only if the ObjectMapper has NOT been provided ( https://github.com/OpenAPITools/openapi-generator/blob/ab803e59e3c97f329800a1d520dea76e9307a792/modules/openapi-generator/src/main/resources/Java/libraries/webclient/ApiClient.mustache#L92 )?

Reply:
Indeed. I would create a default mapper if none is provided and otherwise take the mapper given by the user as is assuming he knows what he does.

Reply:
Then I think we should also move
https://github.com/OpenAPITools/openapi-generator/blob/ab803e59e3c97f329800a1d520dea76e9307a792/modules/openapi-generator/src/main/resources/Java/libraries/webclient/ApiClient.mustache#L131
and
https://github.com/OpenAPITools/openapi-generator/blob/ab803e59e3c97f329800a1d520dea76e9307a792/modules/openapi-generator/src/main/resources/Java/libraries/webclient/ApiClient.mustache#L134

Will push soon.